recipes/xagg/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "xagg" %}
+{% set version = "0.2.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xagg-{{ version }}.tar.gz
+  sha256: b536f73277fb9371741e957680333067bd511033da4d644bfbb8bf8605b67928
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
What's the reason for this not being `noarch: python`? It looks like it's pure Python. If it's just because the build fails on Windows due to a missing dependency, we can ignore that.

Reply:
Yeah. Ok i'll revert back.